tests/tests.toml

```diff
@@ -1000,6 +1000,14 @@ asr = true
 filename = "errors/test_dict14.py"
 run = true
 
+[[test]]
+filename = "errors/test_dict15.py"
+run = true
+
+[[test]]
+filename = "errors/test_dict16.py"
+run = true
+
```

Comment:
Should this be removed?

Reply:
Yes, the earlier PR missed to enable those tests. So, while updating references in this PR, it removed the required references. Hence, I enabled those tests so that we won't affect them.

Related discussion: https://github.com/lcompilers/lpython/pull/2316#discussion_r1319386819

Reply:
Yes, that's fine.